lib/confluence_table.rb

```
@@ -0,0 +1,72 @@
+    page_json = JSON.parse fetch_page
+    current_version = page_json['version']['number']
+    document = Nokogiri::HTML(page_json['body']['storage']['value'])
+    table = document.at('table')
+    row = table.xpath("//tr[td/p[text()='#{project}']]")
+    row.at_css('td:nth-child(2) p').content = pass.to_s
+    row.at_css('td:nth-child(3) p').content = fail.to_s
+    row.at_css('td:nth-child(4) p').content = skipped.to_s
+    row.at_css('td:nth-child(5) p').content = "#{pass_rate.to_s}%"
+
+    update_page(page_json['title'], document, current_version + 1, project)
+  end
```

rubocop

[convention] Assignment Branch Condition size for update_table is too high. [<9, 25, 0> 26.57/23]

(Metrics/AbcSize)